test/parallel/test-http-same-map.js

```diff
@@ -0,0 +1,56 @@
+// Flags: --allow_natives_syntax
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const host = common.localhostIPv4;
+const port = common.PORT;
```

Comment:
> IMHO this should be a blocker for landing any PR, otherwise we could easily end up in the same situation again with cascading test failures if we start being lax about it.

That's reasonable (with the note that there are currently situations that require `common.PORT` or something like it, but you know that and no doubt mean to exclude those situations).